src/components/organisms/AdminVenueView/components/Timing/SpaceTimingPanel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { AdminPanel } from "components/organisms/AdminVenueView/components/AdminPanel";
+import { AdminShowcase } from "components/organisms/AdminVenueView/components/AdminShowcase";
+import { AdminSidebar } from "components/organisms/AdminVenueView/components/AdminSidebar";
+import {
+  AdminSidebarFooter,
+  AdminSidebarFooterProps,
+} from "components/organisms/AdminVenueView/components/AdminSidebarFooter/AdminSidebarFooter";
+import { AdminSidebarTitle } from "components/organisms/AdminVenueView/components/AdminSidebarTitle";
+import { SpaceTimingForm } from "components/organisms/SpaceTimingForm";
+
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import { EventsView } from "../EventsView";
+
+import "./SpaceTimingPanel.scss";
+
+export interface SpaceTimingPanelProps extends AdminSidebarFooterProps {
+  venue: WithId<AnyVenue>;
+}
+
+export const SpaceTimingPanel: React.FC<SpaceTimingPanelProps> = ({
+  venue,
+  ...sidebarFooterProps
+}) => {
+  if (!venue) {
+    return <LoadingPage />;
```

Comment:
Maybe, just kept the logic as it was previously used. 
We didn't have a bug issue although it seems like it's possible some1 to type invalid id in the address bar. 
Generally it's been accessed by click on the tab bar.

My take on this: navigation will probably be refactored next sprint anyway, we can deal with the case then.

Reply:
OK, `SpaceTimingPanel` is only used by one component and it already has handled the problem:

```tsx
  if (!venue) {
    return (
      <WithNavigationBar withSchedule>
        <AdminRestricted>
          <NotFound />
        </AdminRestricted>
      </WithNavigationBar>
    );
  }
```

It can be safely resolved by removing the optionality of the prop.

Reply:
Deleted the `if` @ c2bf6c9a13bd4ed9cafc26a2143a2df5a05ab383